From 300db150dcd6ec1948f7703f0b97bdfc08141008 Mon Sep 17 00:00:00 2001
From: Brian Ryner <bryner@google.com>
Date: Fri, 28 Aug 2020 16:27:29 +1000
Subject: [PATCH 7/7] Merge https://github.com/python/cpython/pull/21588

bugfix, arm64 Mac OS, missing #include <AvailabilityMacros.h>

MAC_OS_X_VERSION_MAX_ALLOWED is defined in this header. Because it is
not included nsexeclength is defined as the wrong type, causing an
incompatible pointer to be passed to _NSGetExecutablePath on arm64
---
 Modules/getpath.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Modules/getpath.c b/Modules/getpath.c
index 092ccc712f..bcc9be7e63 100644
--- a/Modules/getpath.c
+++ b/Modules/getpath.c
@@ -8,6 +8,7 @@
 
 #ifdef __APPLE__
 #include <mach-o/dyld.h>
+#include <AvailabilityMacros.h>
 #endif
 
 /* Search in some common locations for the associated Python libraries.
-- 
2.28.0.402.g5ffc5be6b7-goog

